infrastructure/metadata/infrastructure/testdriver/click_iframe_crossorigin.sub.html.ini

```diff
@@ -2,3 +2,4 @@
   [TestDriver click on a document in an iframe]
     expected:
       if product == "chrome": [PASS, FAIL]  # https://github.com/web-platform-tests/wpt/issues/26295
+      if product == "firefox": [PASS, FAIL]  # https://github.com/web-platform-tests/wpt/issues/45987
```

Comment:
This should also depend on `os == "mac"` so that we're not losing coverage on Linux where I don't think we have any evidence of failures in Firefox.

Reply:
OK, let's do this:

```suggestion
      if os == "mac" and product == "firefox": [PASS, FAIL]  # https://github.com/web-platform-tests/wpt/issues/45987
```